apps/dashboard/src/types/tabler-icons.d.ts

```
@@ -433,4 +433,29 @@ declare module '@tabler/icons-react' {
+  export const IconCircleCheck: Icon;
+  export const IconCircleX: Icon;
+  export const IconCircleDot: Icon;
+  export const IconCirclePlus: Icon;
+  export const IconCircleMinus: Icon;
+  export const IconPlayerPlayFilled: Icon;
+  export const IconPlayerPauseFilled: Icon;
+  export const IconPlayerStopFilled: Icon;
+  export const IconBrain: Icon;
+  export const IconNetwork: Icon;
+  export const IconApi: Icon;
+  export const IconWebhook: Icon;
```

Biome (2.1.2)

[error] Shouldn't redeclare 'IconWebhook'. Consider to delete it or rename it.

'IconWebhook' is defined here:


(lint/suspicious/noRedeclare)